ui/raidboss/data/07-dt/dungeon/ihuykatumu.ts

```diff
@@ -0,0 +1,193 @@
+      type: 'StartsUsing',
+      netRegex: { id: '8E99', source: 'Ihuykatumu Flytrap', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Get under Flytrap',
+        },
+      },
+    },
+    {
+      id: 'Ihuykatumu Prime Punutiy Inhale',
+      type: 'StartsUsing',
+      netRegex: { id: '8E8E', source: 'Prime Punutiy', capture: false },
+      durationSeconds: 10, // prolonged ground damage
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Stay out of inhale',
+        },
+      },
+    },
+    {
+      id: 'Ihuykatumu Prime Punutiy Shore Shaker',
+      type: 'StartsUsing',
+      netRegex: { id: '8EA2', source: 'Prime Punutiy', capture: false },
+      durationSeconds: 6,
+      response: Responses.getOutThenIn(),
+    },
+
+    // ** Drowsie ** //
+    {
+      id: 'Ihuykatumu Drowsie Uppercut',
+      type: 'StartsUsing',
+      netRegex: { id: '98DC', source: 'Drowsie' },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Ihuykatumu Drowsie Wallop Small',
+      type: 'StartsUsing',
+      netRegex: { id: '8E7F', source: 'Ihuykatumu Ivy', capture: false },
+      delaySeconds: 2,
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Dodge Ivy cleaves (small)',
+        },
+      },
+    },
+    {
+      id: 'Ihuykatumu Drowsie Wallop Large',
+      type: 'StartsUsing',
+      netRegex: { id: '8E82', source: 'Ihuykatumu Ivy', capture: false },
+      delaySeconds: 2,
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Dodge Ivy cleaves (big)',
+        },
+      },
+    },
+    {
+      id: 'Ihuykatumu Drowsie Sneeze',
+      type: 'StartsUsing',
+      netRegex: { id: '8E7B', source: 'Drowsie', capture: false },
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'Ihuykatumu Drowsie Flagrant Spread',
+      type: 'HeadMarker',
+      netRegex: { id: '008B' },
+      condition: Conditions.targetIsYou(),
+      response: Responses.moveAway(),
+    },
+
+    // ** Apollyon ** //
+    {
+      id: 'Ihuykatumu Apollyon Blade',
+      type: 'StartsUsing',
+      netRegex: { id: ['8DFB', '8E04'], source: 'Apollyon' },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Ihuykatumu Apollyon High Wind',
+      type: 'StartsUsing',
+      netRegex: { id: '8DF5', source: 'Apollyon', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Ihuykatumu Apollyon Swarming Locust First Cleaves',
+      type: 'Ability',
+      netRegex: { id: '8DF7', source: 'Apollyon', capture: false },
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Get away after 3rd jump',
```

Comment:
```suggestion
          en: Away from 3rd jump',
```
No strong feelings on this, suggestion only. Maybe "away on 3rd jump" could work too.